index.bs

```diff
@@ -152,117 +152,111 @@ mean the same thing as:
+The Loader constructor is the %Loader% intrinsic object and the initial value of the <b>Loader</b> property of the the Reflect object. When called as a constructor it creates and initializes a new Loader object. When <b>Loader</b> is called as a function rather than as a constructor, it throws an exception.
 
-The Reflect.Loader constructor is designed to be subclassable. It may be used as the value in an extends clause of a class definition. Subclass constructors that intend to inherit the specified Loader behaviour must include a super call to the Reflect.Loader constructor to create and initialize the subclass instance with the internal stage necessary to support the Reflect.Loader.prototype built-in methods.
+The <b>Loader</b> constructor is designed to be subclassable. It may be used as the value of an <b>extends</b> clause of a class definition. Subclass constructors that intend to inherit the specified <b>Loader</b> behaviour must include a <b>super</b> call to the <b>Loader</b> constructor to create and initialize the subclass instance with the corresponding internal slots.
 
-<h4 id="new-reflect-loader">Reflect.Loader()</h4>
+<h4 id="new-loader">Loader()</h4>
 
-When Reflect.Loader is called with no arguments, the following steps are taken:
+When Loader is called with no arguments, the following steps are taken:
 
 <emu-alg>
 1. If NewTarget is *undefined*, then throw a *TypeError* exception.
-1. Let _O_ be OrdinaryCreateFromConstructor(NewTarget, "Reflect.Loader.prototype").
-1. ReturnIfAbrupt(_O_).
-1. Let _registry_ to a new Registry(_O_).
-1. Set _O_’s [[Registry]] internal slot to _registry_.
+1. Let _O_ be ? OrdinaryCreateFromConstructor(NewTarget, "%LoaderPrototype%", «[[Realm]], [[Registry]]»).
+1. Set _O_’s [[Registry]] internal slot to CreateRegistry().
 1. Return _O_.
 </emu-alg>
 
-<h3 id="properties-of-the-loader-constructor">Properties of the Reflect.Loader Constructor</h3>
+<h3 id="properties-of-the-loader-constructor">Properties of the Loader Constructor</h3>
 
-The value of the \[[Prototype]] internal slot of the Reflect.Loader constructor is the intrinsic object %FunctionPrototype%.
+The value of the \[[Prototype]] internal slot of the Loader constructor is the intrinsic object %FunctionPrototype%.
 
-Besides the internal slots and the length property (whose value is 0), the Reflect.Loader constructor has the following properties:
+Besides the internal slots and the length property (whose value is 0), the Loader constructor has the following properties:
 
-<h4 id="Reflect.Loader.prototype">Reflect.Loader.prototype</h4>
+<h4 id="loader-prototype">Loader.prototype</h4>
 
-The value of Reflect.Loader.prototype is an ordinary object.
+The value of Loader.prototype is %LoaderPrototype%.
```

Comment:
The initial value of Loader.prototype is the intrinsic object %LoaderPrototype%.